api/envoy/extensions/http/injected_credentials/oauth2/v3/oauth2.proto

```diff
@@ -18,12 +20,10 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 option (xds.annotations.v3.file_status).work_in_progress = true;
 
 // [#protodoc-title: OAuth2 Credential]
-// [#not-implemented-hide:]
 // [#extension: envoy.http.injected_credentials.oauth2]
 
 // OAuth2 extension can be used to retrieve an OAuth2 access token from an authorization server and inject it into the
 // proxied requests.
-// Currently, only the Client Credentials Grant flow is supported.
```

Comment:
Is this following the changes in this PR, or is it something from a prior PR?
(want to ensure that I understand the PR correctly).

Reply:
my bad. reverted!